update-dependencies.js

```diff
@@ -0,0 +1,211 @@
+'use strict';
+
+require('shelljs/global');
+
+const version = exec('node --version', {silent:true});
+
+console.log('node version: ', version.stdout);
+
+const installBower = exec('npm install -g bower');
+
+const prepBranch = (project, release, lightningThemeRelease) => {
+	return new Promise((resolve, reject) => {
+		const execOpts = {
+			silent: true,
+			async: false,
+			shell: '/bin/bash'
+		};
+
+		const cd = `cd ../${project}`
+		const gitstuff = `git stash && git checkout master && git pull upstream master && git status`
```

Comment:
feature assumptions:
- parent project on "upstream" origin
- primary branch is "master"

could be modified to use a prompt but not necessary for first pass